dashboard/app/views/levels/_pixelation.html.haml

```diff
@@ -0,0 +1,70 @@
+= stylesheet_link_tag 'levels/pixelation'
+
+#visualizationColumnHeader.responsive.hide_on_v1
+  #file_format
+    %p#file_format_title
+      %strong= I18n.t('pixelation.image_file_format_header')
+    .byte= I18n.t('pixelation.image_file_format_width')
+    .byte= I18n.t('pixelation.image_file_format_height')
+    .byte.hide_on_v1.hide_on_v2= I18n.t('pixelation.image_file_format_bits_per_pixel')
+    %div
+      = I18n.t('pixelation.image_file_format_data_a')
+      %p.hide_on_v2= I18n.t('pixelation.image_file_format_data_b')
+#visualizationEditorHeader.responsive
+  %table#image_controls
+    %tr
+      %td
+        %label{for: 'width'}= I18n.t('pixelation.image_width')
+      %td
+        %input#width{type: 'number', min: '1', max: '255', value: '10', oninput: 'setSliders()', onchange: 'setMinTextValues()', disabled: true}
+      %td.hide_on_v2
+        %input#widthRange.rangeInput{type: 'range', min: '1', max: '255', value: '10', oninput: 'changeVal("width")', onchange: 'changeVal("width")', disabled: true}
+    %tr
+      %td
+        %label{for: 'height'}= I18n.t('pixelation.image_height')
+      %td
+        %input#height{type: 'number', min: '1', max: '255', value: '10', oninput: 'setSliders()', onchange: 'setMinTextValues()', disabled: true}
+      %td.hide_on_v2
+        %input#heightRange.rangeInput{type: 'range', min: '1', max: '255', value: '10', oninput: 'changeVal("height")', onchange: 'changeVal("width")', disabled: true}
+    %tr.hide_on_v1.hide_on_v2
+      %td
+        %label{for: 'bitsPerPixel'}= I18n.t('pixelation.bits_per_pixel_header')
+      %td
+        %input#bitsPerPixel{type: 'number', min: '1', max: '24', value: '1', oninput: 'setSliders()', onchange: 'setMinTextValues()', disabled: true}
+      %td
+        %input#bitsPerPixelSlider.rangeInput{type: 'range', min: '1', max: '24', oninput: 'changeVal("bitsPerPixel")', onchange: 'changeVal("width")', disabled: true}
+  %label.right.hide_on_v1.hide_on_v2
+    = I18n.t('pixelation.pixel_format_header')
+    #pixel_format
+  .label_group.encoding_controls
+    %label
+      = I18n.t('pixelation.binary')
```

Comment:
These radio buttons are side by side. I wonder how it looks when you do a LTR language.

Reply:
Since they are side by side, I feel like they would look better if they had an outline/container around the text + radio.